lib/workers/dependency.rb

```diff
@@ -35,7 +35,9 @@ def get_results(dependencies)
       dependencies.select! {|dep| @dependency_types.include?(dep.first.type)}
       dependencies.each do |dep, latest_version|
         dependency_hash = dependency(dep.name, dep.requirement.to_s, latest_version.to_s, dep.type)
-        if latest_version.nil? || dependency_hash[:required] == 'unknown'
+        if dependency_hash[:required] == 'unknown'
+          results[:no_requirement] << dependency_hash
```

Comment:
I thought it would make sense to distinguish between:
1. Dependencies that didn't have a version requirement set in the Pipfile/Cargofile (for instance: git and local dependencies)
2. Dependencies that did have a version requirement set, but we couldn't find the latest version

1 is now going into the `:no_requirement` basket, and 2 into the `:unknown` basket. Make sense?

Reply:
Makes sense!